### PR TITLE
Fix/send config

### PR DESCRIPTION
**Pull request by trap98**

### CODE REVIEW

File: src/core/CoolBoard.cpp [hunk 1]
```diff
@@ -91,8 +91,13 @@ void CoolBoard::loop() {
   this->connect();
   INFO_LOG("Updating RTC...");
   this->rtc.update();
-  if (!this->sleepActive) {
+  if (!SPIFFS.exists("/configSent.flag")) {
     sendAllConfig();
+    File f;
+    if (!(f = SPIFFS.open("/configSent.flag", "w"))) {
+      ERROR_LOG("Can't create file configSent.flag in SIFFS");
```
Comment (3rwww1) [inline src/core/CoolBoard.cpp]: `SPIFFS`, not `SIFFS`

File: src/core/CoolBoard.cpp [hunk 2]
```diff
@@ -91,8 +91,13 @@ void CoolBoard::loop() {
...
+    }
```
Comment (3rwww1) [inline src/core/CoolBoard.cpp]: put that in a `else` block

**Review by 3rwww1 — CHANGES_REQUESTED** [review]

File: src/core/CoolBoard.cpp (hunk 2 above)
Comment (Kameeno) [reply src/core/CoolBoard.cpp]: its not better to do: 
```
if(! SPIFFS.exists("/configSent.flag") ){
f = SPIFFS.open("/configSent.flag", "w");
f.close();
}```
?

Comment (3rwww1) [reply src/core/CoolBoard.cpp]: That's exactly what he does @Kameeno :)

**Review by 3rwww1 — APPROVED** [review]

**Comment by Kameeno** [comment]
```  if (!SPIFFS.exists("/configSent.flag")) {
    sendAllConfig();
    File f;
    if (!(f = SPIFFS.open("/configSent.flag", "w"))) {
      ERROR_LOG("Can't create file configSent.flag in SPIFFS");
    } else {
      f.close();
    }
  }
  INFO_LOG("Listening to saved messages...");
  this->mqttListen();
  if (CoolFileSystem::hasSavedLogs()) {
    INFO_LOG("Sending saved messages...");
    this->sendSavedMessages();
  }
  DynamicJsonBuffer jsonBuffer;
  JsonObject &root = jsonBuffer.createObject();
  JsonObject &state = root.createNestedObject("state");
  JsonObject &reported = state.createNestedObject("reported");
  INFO_LOG("Collecting board and sensor data...");
  this->readPublicIP(reported);
  this->readBoardData(reported);
  this->readSensors(reported);
```

if we send allConfig one time, why we read and report to the json again the config and the public ip ?

**Pull request merged**